tests/unit/test_utils.py

```diff
@@ -2,26 +2,28 @@
 
 import pytest
 
-from circuit_maintenance_parser.utils import city_timezone
+from circuit_maintenance_parser.utils import Geolocator
+
+geolocator = Geolocator()
 
 
-# TODO: Recover testing when issue #101 is fixed
 @pytest.mark.parametrize(
     "city, timezone",
     [
-        # ("North Bergen, NJ", "America/New_York"),
-        # Removing some tests to speed up testing
-        # ("Carlstadt, NJ", "America/New_York"),
-        # ("Atlanta, GA", "America/New_York"),
-        # ("New York, NY", "America/New_York"),
-        # ("Worcester, MA", "America/New_York"),
-        # ("Salt Lake City, Utah", "America/Denver"),
-        # ("Los Angeles, CA", "America/Los_Angeles"),
-        # ("Dublin, Ireland", "Europe/Dublin"),
-        # ("Barcelona, Spain", "Europe/Madrid"),
-        # ("Sydney, Australia", "Australia/Sydney"),
+        ("North Bergen, NJ", "America/New_York"),
+        ("Carlstadt, NJ", "America/New_York"),
+        ("Atlanta, GA", "America/New_York"),
+        ("New York, NY", "America/New_York"),
+        ("Worcester, MA", "America/New_York"),
+        ("Salt Lake City, Utah", "America/Denver"),
+        ("Los Angeles, CA", "America/Los_Angeles"),
+        ("Dublin, Ireland", "Europe/Dublin"),
+        ("Barcelona, Spain", "Europe/Madrid"),
+        ("Sydney, Australia", "Australia/Sydney"),
+        ("Guadalajara, Spain", "Europe/Madrid"),
+        ("Guadalajara, Mexico", "America/Mexico_City"),
```

Comment:
Just confirming - are these a mix of "city + country matches CSV" and "city alone matches CSV" so that we are testing both cases?

Reply:
exactly, this test was to validate that case, we try to get the city + country first to avoid ambiguities